.eslintrc.js

```diff
@@ -1,5 +1,8 @@
 process.env.ENABLE_NEW_JSX_TRANSFORM = 'true';
 
+/**
+ * @type {import("eslint").Linter.Config}
+ */
```

Comment:
Slightly unrelated but as I discovered that we can use the JSDoc `@type` to provide type declarations for an object (and thus intellisense), I've added this in most of the config files of the repo.

Reply:
Nice!